### DISCUSSION TITLE
Bug or feature request??csv files are imported as attachment files not tables??  also i am not able to paste tables ? do i need to do something special to do that?

### DISCUSSION THREAD

**Discussion by SHx7ejs7fCrP** [TriliumNext/Trilium#5839] [Q&A] [answered]
I am trying to -move from notion to trillium. Notion exports data in markdown and csv files. 
It has a table like database format where each field of the table is a separate markdown file
And its export contains a csv of the said database table as a whole and separate markdown files for each cell/page . 
While markdown file is imported as a note, the csv file is simply imported as an attachment ..
Is there any way of populating a table with a csv  flle??
I don''t mind if i have to do something in between like open an excel file and import csv file into it first. 
The ultimate aim is to create a table and in the cell of table add inline link to the each page.
Also other exporting way supported by notion is html
Would they be better to import into notion???
I have a lot of data and if there is a long way of doing multiple steps, i dont mind it ..
Thanks
Also here zadam mentioned that i should be able to paste tables
I am not able to do so..
https://github.com/zadam/trilium/issues/175
Is there something i need to do over and above native install of trilium to be able to do that?

**Comment by zadam** [comment, marked as answer]
Hi, CSV transformation into a table is not supported.

Tables in HTML should work fine though ...

**Comment by plashal** [comment]
I hear you, "should" be fine, but it'd be nice to have them in a native format.

**Comment by meichthys** [comment]
[Trilium has entered maintenance mode](https://github.com/zadam/trilium/issues/4620#issuecomment-2094495698). Future enhancements will be addressed in TrilumNext: https://github.com/TriliumNext/trilium/issues/4980
